Themes/default/css/index.css

```diff
@@ -1726,13 +1726,20 @@ tr.windowbg td, tr.bg td, .table_grid tr td {
 	background: url(../images/icons/main_icons_sprite.png) no-repeat -5px -5px / 260px auto;
 	vertical-align: middle;
 }
+.main_icons.alerts::before {
+	background: url(../images/icons/bell.png);
```

Comment:
Should we use this bell icon for the Notification settings in the user drop down menu?

Reply:
I'd be open to that change. What do you think, @DiegoAndresCortes?

Reply:
Yes, makes sense and alerts are usually first choice

Reply:
Done.